src/net.h

```diff
@@ -1126,6 +1127,20 @@ class CConnman
      */
     std::vector<CService> m_onion_binds;
 
+    /**
+     * Mutex protecting m_i2p_sam_sessions.
```

Comment:
```suggestion
     * Mutex protecting m_i2p_sessions.
```
-----
The role of this mutex and of the queue it guards might be more clearly named something like `m_unused_i2p_sessions` and `m_unused_i2p_sessions_mutex`.

Reply:
Renamed.